.circleci/config.yml

```diff
@@ -1121,6 +1120,13 @@ jobs:
                 ]
               }
 
+  deploy-to-spm:
+    docker:
+      - image: cimg/base:stable
+    resource_class: small
+    steps:
+      - push-to-spm
```

Comment:
nitpick, but since the `push-to-spm` command is only used in this job, I think we could just have it here directly, without the need of a command:

```
deploy-to-spm:
    docker:
      - image: cimg/base:stable
    resource_class: small
    steps:
      - checkout
      - setup-git-credentials
      - run:
          name: Clone purchases-ios and push to purchases-ios-spm
          command: |
            git clone https://github.com/RevenueCat/purchases-ios.git
            cd purchases-ios
            git fetch --tags
            git remote set-url origin https://github.com/RevenueCat/purchases-ios-spm.git
            git push origin
            git push --tags
 ```

Reply:
Done!